### PR TITLE
add tests occ log:manage

### PR DESCRIPTION
**Pull request by davitol**
Add 1st tests for https://github.com/owncloud/core/issues/33136

## How it was tested

Running test in local and passed.
 ```sudo -u www-data ./run.sh --feature features/cliMain/logManage.feature```

### CODE REVIEW

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 1]
```diff
@@ -461,6 +461,32 @@ public function theDisplayNameReturnedByTheOccCommandShouldBe($displayName) {
 		PHPUnit_Framework_Assert::assertEquals($displayName, $lastOutputDisplayName);
 	}
 
+	/**
+	 * @When administrator sets the log level to :level using the occ command
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: When the administrator..

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 2]
```diff
@@ -461,6 +461,32 @@ public function theDisplayNameReturnedByTheOccCommandShouldBe($displayName) {
...
+	 * @return void
+	 */
+	public function theAdministratorSetsLogLevelUsingTheOccCommand($level) {
+		$this->featureContext->invokingTheCommand(
+			"log:manage --level $level"
+		);
+	}
+
+	/**
+	 * @When administrator sets the timezone to :timezone using the occ command
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: When the administrator..

File: tests/acceptance/features/cliMain/logManage.feature [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    When administrator sets the log level to "error" using the occ command
+    Then the command should have been successful
+    And the command output should contain the text 'Log level: Error'
+
+  Scenario: Admin set the log level to Fatal
+    When administrator sets the log level to "fatal" using the occ command
+    Then the command should have been successful
+    And the command output should contain the text 'Log level: Fatal'
+
+  Scenario: Admin set the log level to Warning
```
Comment (phil-davis) [inline tests/acceptance/features/cliMain/logManage.feature]: This scenario is already above.

**Review by phil-davis — COMMENTED** [review]
See comments.

Also you need to add an entry to the drone matrix so that it runs  the new test suite.

**Review by phil-davis — APPROVED** [review]
LGTM

**Comment by davitol** [comment]
Backport here: https://github.com/owncloud/core/pull/33210

**Pull request merged**